try:
    numerator = 10
    denominator = 0

    result = numerator/denominator

    print(result)
except:
    print("Error: Denominator cannot be 0.")




try:
  print(x)
except:
  print("Something went wrong")
finally:
  print("The 'try except' is finished")


